eslint.config.mjs

```diff
@@ -0,0 +1,95 @@
+import typescriptEslint from '@typescript-eslint/eslint-plugin';
+import globals from 'globals';
+import tsParser from '@typescript-eslint/parser';
+import svelteParser from 'svelte-eslint-parser';
+import js from '@eslint/js';
+
+export default [
+	js.configs.recommended,
+	{
+		ignores: [
+			'**/node_modules/**',
+			'build/**',
+			'.svelte-kit/**',
+			'package/**',
+			'**/*.d.ts',
+			'.vercel/**'
+		]
+	},
+	{
+		files: ['**/*.{js,ts,svelte}'],
+		plugins: {
+			'@typescript-eslint': typescriptEslint
+		},
+		languageOptions: {
+			globals: {
+				...globals.browser,
+				...globals.node,
+				__VER__: 'readonly'
+			},
+			parser: tsParser,
+			parserOptions: {
+				ecmaVersion: 2020,
+				sourceType: 'module',
+				extraFileExtensions: ['.svelte']
+			}
+		},
+		rules: {
+			'@typescript-eslint/ban-ts-comment': 'error',
+			'@typescript-eslint/naming-convention': [
+				'error',
+				{
+					selector: 'default',
+					format: null
+				},
+				{
+					selector: 'variable',
+					format: ['snake_case'],
+					leadingUnderscore: 'allow',
+					trailingUnderscore: 'allow'
+				},
+				{
+					selector: 'variable',
+					modifiers: ['const'],
+					format: ['UPPER_CASE']
```

Comment:
Not a fan of this rule since we use `const` by default for variables. _Actual_ constants make sense to be all uppercase, but that rule should be limited to a specific file / folder where constants are defined.

Reply:
Yeah I was wavering on this one as well. Maybe the rule should be that const can be snake or SHOUTING_SNAKE so that we can use UPPER for true constants.

Reply:
changed to 
```
selector: 'variable',
modifiers: ['const'],
format: ['snake_case', 'UPPER_CASE']
```